recipes/ros-conda-base/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: ros-conda-base
+  version: 0.0.1
+
+build:
+  number: 0
+
+dependencies:
+  run:
+    - ros-conda-mutex * melodic
+  run_constrained:
+    - boost 1.68.*
+    - cmake >=3.7.2
+    # - gazebo >=9.0
```

Comment:
Why are these commented?

Reply:
some don't exist yet (e.g. gazebo, ogre3d) and others are not yet required. But i think you're right, I should add the ones that exist already.

Reply:
I'm fairly sure they don't need to exist to be listed here and putting them in now avoids having bad metadata somewhere in the history.